source/HailPanel.cpp

```diff
@@ -61,17 +61,8 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	// Update default hail responses based on the hailed ship's government and status condition.
 	else if(gov->IsEnemy())
-	{
-		if(!ship->IsDisabled())
```

Comment:
Since you removed this check you can now bribe disabled ships. You should keep it imo